tests/PHPStan/Type/TypeCombinatorTest.php

```diff
@@ -3507,6 +3507,22 @@ public function dataIntersect(): iterable
 				StrictMixedType::class,
 				'mixed',
 			],
+			[
+				[
+					new NeverType(true),
+					new IntegerType(),
+				],
+				NeverType::class,
+				'*NEVER*=explicit',
+			],
```

Comment:
here we have a test which now returns a explicit NeverType. all other tests where adjusted to compensate the additional `implicit` information in the description. no changes regarding the typtes itself.